### DISCUSSION TITLE
Unable to use the application

### DISCUSSION THREAD

**Discussion by Mickael81** [Akylas/alpimaps#52] [Q&A]
Hello,
your app doesn't seem to work. As you can see from the attached screenshots, when I open it, I don't have any maps. I can't locate myself. How can I fix this?
![Screenshot_20241218_203041_Alpi Maps](https://github.com/user-attachments/assets/3db97851-04a4-4a6b-909e-35490a3e8f1d)
![Screenshot_20241218_203052_Alpi Maps](https://github.com/user-attachments/assets/2713b711-b1b7-4905-8491-85d7b92cf0ea)

**Comment by farfromrefug** [comment]
@Mickael81 sorry it is a bug. Which version are you using ?

**Reply by Mickael81** [reply]
@farfromrefug I am using version 2.8.7. This is the latest version released.

**Reply by farfromrefug** [reply]
@Mickael81 Ok. If you clear app dats and restart ? It should shoe you 2 dialogs from first opening then it should show you OpenStreetMap raster tiles. 
Let me.know If you still see the getOptions error. 
Can you share your phone specs BTW ?

**Comment by Mickael81** [comment]
@farfromrefug 
![Screenshot_20241219_102643_Alpi Maps](https://github.com/user-attachments/assets/830ece61-1487-4169-83a6-abe8cbcd44e7)
![Screenshot_20241219_102534_Alpi Maps](https://github.com/user-attachments/assets/63457311-8844-42b0-b99c-f56fff6d3ff4)
![Screenshot_20241219_102546_Alpi Maps](https://github.com/user-attachments/assets/26bd89d4-888a-4e81-b393-30fc76d79d74)
![Screenshot_20241219_102452_Alpi Maps](https://github.com/user-attachments/assets/91a5f241-d70d-4e83-a108-a2321f31171c)
when opening the application, I have no dialog box that appears on the screen. I arrive directly on a grid screen like on the first screenshot that I attached to you yesterday. 
I tried several things like uninstalling and reinstalling the application, clearing the cache and data of the application but unfortunately, when starting I always come across a screen without any dialog box. 
The error message appears after touching the button located at the bottom left of the screen, then "+", then selecting a card type in the list. The error is identical when I select another card in this same list. 
By touching the other button at the bottom left of the screen located just above, I arrive at the screen with a page called "Articles" with 2 tabs ("Itineraries" and "Markers") but nothing appears on the screen. 
I have a Samsung Galaxy A54 5G (OneUI 6.1 with Android 14)

**Reply by farfromrefug** [reply]
@Mickael81 thanks for the clear explanation. I found the issue. Last releases are broken :s. Will fix it as soon as possible

**Comment by Mickael81** [comment]
@farfromrefug Thank you very much. 👍 I recently discovered your app on GitHub. So I'm looking forward to using it for hiking. I'll try to improve the translation on Weblate when the app is up and running again. 😉

**Reply by farfromrefug** [reply]
@Mickael81 great ! BTW if you have some space on a sdcard i can share the full france data which will allow you to do everything without internet!

**Comment by Mickael81** [comment]
@farfromrefug Oh yes! I am very interested. 😃 I have a 128GB SD card on my phone. I have 50GB left but I can free up space if needed. Can you send it to me?

**Reply by farfromrefug** [reply]
@Mickael81 all good should be around 8Gb. Contact me through the form on https://www.akylas.fr and i ll share a link for you

**Comment by Mickael81** [comment]
@farfromrefug Thanks a lot. I sent you a message via the form you provided to me.
